src/modules/subscription/reducer.spec.ts

```diff
@@ -47,14 +58,52 @@ describe('when reducing the get subscription success action', () => {
       subscriptionDetails: { ...objectToCamel(subscription.details), ignoreAllInApp: true }
     }
   })
+  describe('and there is no email nor unconfirmedEmail', () => {
+    it('should return a state with the default subscription details set without the email and the loading state cleared', () => {
```

Comment:
Why is it the default subscription details? It's setting the subscription details that came with the action.

Reply:
can you check again?